transport.go

```diff
@@ -49,6 +50,20 @@ type accessToken struct {
 	Repositories []github.Repository            `json:"repositories,omitempty"`
 }
 
+// RefreshTokenError represents a custom error for the refresh access tokens operations.
+// It enables the caller to inspect the root cause and response.
+type RefreshTokenError struct {
+	Message        string
+	RootCause      error
+	InstallationID int64
+	Request        *http.Request
+	Response       *http.Response
+}
+
+func (e RefreshTokenError) Error() string {
```

Comment:
I think we want a pointer here, since most users will expect the type to be nil-able (e.g. `if err != nil ...`)

```suggestion
func (e *RefreshTokenError) Error() string {
```

Reply:
Fixed in https://github.com/bradleyfalzon/ghinstallation/pull/45/commits/557bf757fc38e8ed140e4edea47d3069af8af3b1